ci/shared.sh

```diff
@@ -112,6 +112,17 @@ _run_cucumber_tests() {
     fi
   done
 
+  # Generate authn_local socket ENV variables based on the
+  # number of parallel processes.
+  for (( i=1; i <= ${#parallel_services[@]}; ++i )); do
+    index=$(( i - 1 ))
```

Comment:
`index` unused?

Also if I'm not mistaken you could simplify the beginning of the for loop to:
`for i in "${!parallel_services[@]}"; do`

Reply:
> index unused?

Yep - good catch.

> Also if I'm not mistaken you could simplify the beginning of the for loop to:
`for i in "${!parallel_services[@]}"; do`

You're right, but we're using a counting-style 1-based indexing for the parallel services. The updated loop would start from 0, so I think I'll leave it as-is.